taxi/views.py

```diff
@@ -110,9 +155,7 @@ class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
 @login_required
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
```

Comment:
In the `toggle_assign_to_car` function, you are fetching the driver using `request.user.id`. Ensure that the `Driver` model is linked with the user model correctly. If `Driver` is a separate model, you might need to fetch the driver differently.